### DISCUSSION TITLE
New action hook for batch item update

### DISCUSSION THREAD

**Discussion by patvong** [directus/directus#2869] [Feature Requests]
# Feature Request
Add a new [hook action type](https://docs.directus.io/extensions/hooks.html#action-hooks): `batch.update`.

## What problem does this feature solve?
It would allow the ability to create hooks on the batch edit action so we can run custom code.
My use case is that I wanna prevent a batch update on a specific field of a collection.

## How do you think this should be implemented?
Expose new action hook types:
*  `batch.update`
* `batch.update:before`
* `batch.update:after`

The data passed in the hooks would be the list of items that are modified in a batch.

## Open questions
What would be the high-level effort to implement this? 
Our team would be interested to work on this if it's a low priority for the core Directus team. Although we would need some guidance on this.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
